Documentation/Introduction/Index.rst

```diff
@@ -25,6 +25,16 @@ The extension provides a new content element called "Media Consent Opt-In" which
 has two specific fields: one for the HTML snippet embedding the content, another
 for selecting the content provider (Facebook, Twitter...)
 
+..  _template-layout:
+
+Template Layout
+===============
+
+=======
```

Comment:
What is that? Is that an empty header?

Reply:
I fixed a conflict this line was part of that I assume. I removed this